tools/archetype/binding/src/main/resources/archetype-resources/ESH-INF/binding/binding.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="${bindingId}"
-				 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
Did you change the tabs to spaces on purpose in this file? Afaik, our auto-formatter uses tabs in XML files and hence we should also use the same in the initially generated files.